cosmos/tucana_712.json

```diff
@@ -0,0 +1,56 @@
+{
+  "rpc": "https://rpc.birdee-2.tucana.zone",
+  "rest": "https://lcd.birdee-2.tucana.zone",
+  "chainId": "tucana_712-1",
+  "chainName": "Tucana Testnet",
+  "evm": {
+    "chainId": 712,
+    "rpc": "https://evm-rpc.birdee-2.tucana.zone"
+  },
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/tucana_712/chain.png",
+  "walletUrlForStaking": "https://testnet.keplr.app/chains/tucana",
```

Comment:
Hi. could you remove this line? The page isn't dynamically generated.

Reply:
removed, thanks for reminding